src/components/ArticulosCientificos/styles.scss

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

Comment:
read this article to improve the approach for importing the fonts https://nextjs.org/docs/pages/building-your-application/optimizing/fonts